core/templates/pages/volunteer-page/volunteer-page.component.ts

```diff
@@ -157,77 +174,152 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
-            'reading and writing skills!“'),
-          name: '- Wala Awad',
-          type: 'Learner Feedback Program Volunteer, Palestine'
+            '“I contribute to Oppia because seeing the community ' +
+            'and the impact it creates makes me hopeful for the future of ' +
+            'education, and I want to be a part of that change.“'),
+          name: '- Diana Chen',
+          type: 'Product Manager'
         }
       ]
     };
 
     this.lessonCreation = {
       images: [
+        '/volunteer/profile_images/aanuoluwapo.jpg',
+        '/volunteer/profile_images/viksar.jpg',
+        '/volunteer/profile_images/jackson.jpg',
+      ],
+      caption: [
+        {
+          content: (
+            '“At Oppia, my love for writing stories as a ' +
+            'creative intertwines with my passion for ' +
+            'education. “'),
+          name: '- Aanuoluwapo Adeoti',
+          type: 'Lessons team'
+        },
+        {
+          content: (
+            '“Contributing to Oppia has been incredibly rewarding. ' +
+            'I\'ve always loved helping others, so being able to use ' +
+            'such a great platform to provide quality education to ' +
+            'those in need is truly fulfilling.“'),
+          name: '- Viksar Dubey',
+          type: 'Practice questions team'
+        },
+        {
+          content: (
+            '“I feel so grateful to help children gain access to ' +
+            'educational resources. I feel proud when I contribute, ' +
+            'and excited when my team makes progress.“'),
+          name: '- Christopher Jackson Felton',
+          type: 'Lessons team'
+        }
+      ]
+    };
+
+    this.translation = {
+      images: [
+        '/volunteer/profile_images/giovana.jpg',
+        '/volunteer/profile_images/kanupriya.jpg',
+        '/volunteer/profile_images/vanessa.jpg',
+        '/volunteer/profile_images/pretty.jpg',
         '/volunteer/profile_images/anubhuti.jpg',
-        '/volunteer/profile_images/anmol.jpg',
-        '/volunteer/profile_images/nitya.jpg',
       ],
       caption: [
         {
           content: (
-            '“Volunteering with Oppia gives me immense ' +
-            'satisfaction because it aligns with a tenet ' +
-            'I firmly believe in: everyone should have access to basic ' +
-            'education. I\'m glad to be a part of the organization!“'),
-          name: '- Anubhuti Varshney',
-          type: 'Translations/Voiceovers Coordinator'
+            '“I was always told that quality education is only for those who ' +
+            'are privileged. But by volunteering at Oppia, I prove every day ' +
+            'hat a decent education is for everyone and I feel grateful to ' +
+            ' be able to help thousands of people worldwide on this journey.“'),
+          name: '- Giovana Alonso',
+          type: 'Brazilian, Portuguese Translator'
```

Comment:
Drop the comma. "Brazilian Portuguese" is one language.

Reply:
Done